Lib/_pyio.py

```diff
@@ -40,6 +40,28 @@
 _CHECK_ERRORS = _IOBASE_EMITS_UNRAISABLE
 
 
+def text_encoding(encoding, stacklevel=2):
+    """
+    Helper function to choose the text encoding.
+
+    When encoding is not None, just return it.
+    Otherwise, return the default text encoding (i.e. "locale").
+
+    This function emits EncodingWarning if *encoding* is None and
+    sys.flags.warn_default_encoding is true.
+
+    This function can be used in APIs having encoding=None option.
+    But please consider encoding="utf-8" for new APIs.
+    """
+    if encoding is None:
+        encoding = "locale"
+        if sys.flags.warn_default_encoding:
+            import warnings
+            warnings.warn("'encoding' argument is not specified.",
+                          EncodingWarning, stacklevel + 1)
```

Comment:
```suggestion
            warnings.warn("'encoding' argument not specified.",
                          EncodingWarning, stacklevel + 1)
```

Briefer form, just fix the tense and inclusion of "is" not matching the omission of the definite article "the"

```suggestion
            warnings.warn(
                "The 'encoding' argument was omitted; please "
                "specify one (e.g. 'utf-8'), or explicitly pass 'locale' "
                "to use the system-dependent current locale encoding",
                EncodingWarning, stacklevel + 2)
```

More complete form, giving advice for how to resolve the warning, if that's appropriate here.